.jenkins/jenkins.groovy

```diff
@@ -28,17 +28,32 @@ def buildStrimziImages() {
     """)
 }
 
-def runSystemTests(String workspace, String testCases, String testProfile, String excludeGroups) {
-    withMaven(mavenOpts: '-Djansi.force=true') {
-        sh(script: "mvn -f ${workspace}/systemtest/pom.xml -P all verify " +
-                "-Dgroups=${testProfile} " +
-                "-DexcludedGroups=${excludeGroups} " +
-                "-Dit.test=${testCases} " +
-                "-Djava.net.preferIPv4Stack=true " +
-                "-DtrimStackTrace=false " +
-                "-Dstyle.color=always " +
-                "--no-transfer-progress " +
-                "-Dfailsafe.rerunFailingTestsCount=2")
+def runSystemTests(String workspace, String testCases, String testProfile, String excludeGroups, String testsInParallel) {
+
+    def runTestsCommand = "mvn -f ${workspace}/systemtest/pom.xml -P all verify " +
+            "-Dgroups=${testProfile} " +
+            "-DexcludedGroups=${excludeGroups} " +
+            "-Dit.test=${testCases} " +
+            "-Djava.net.preferIPv4Stack=true " +
+            "-DtrimStackTrace=false " +
+            "-Dstyle.color=always " +
+            "--no-transfer-progress " +
+            "-Dfailsafe.rerunFailingTestsCount=2"
+
+    // default (sequence mode)
+    if (testsInParallel == "1") {
+        withMaven(mavenOpts: '-Djansi.force=true') {
```

Comment:
Is it possible to add the `-Djansi.force=true` to `runTestsCommand` and then just
```
withMaven() {
...
}
```
? Or is it non-sense? (just to have it on one place)

Reply:
I think I can refactor without diving seq and parallel mode. Because when you specify parallel=1 (it will be basically seq mode).

Reply:
So I will rewrite it.

Reply:
Okay, that sounds good. Thanks :)